pkg/config/application.go

```diff
@@ -644,3 +646,28 @@ func (c *DeploymentChainTriggerCondition) Validate() error {
 	}
 	return nil
 }
+
+func LoadApplication(repoPath string, app *model.Application) (*GenericApplicationSpec, error) {
```

Comment:
How about this?
```suggestion
func LoadApplicationConfig(repoPath, configRelPath string) (*GenericApplicationSpec, error) {
```

Reply:
I think this is fine because this function is under the config package and KIND is also checked internally.

Reply:
I get your point. How about the change of parameter? The whole app structure is not needed actually.

Reply:
Alright, addressed by https://github.com/pipe-cd/pipecd/pull/3730/commits/8ed9b3b92dabe27f4e6c506d6880caf2c7f9c92d.

Reply:
@knanao Thank you.

I knew that the package name is `config` so we can remove the `Config` suffix from the function name.
But, I personally feel a bit strange with the name `LoadApplication`.

I will still be happy if any of them get picked. 👍  Just want to know how you guys feel with that name @khanhtc1202 @knanao .

Reply:
In my opinion, I think these function names should have regularity in the project. Since there were already some functions that did not already have Config in the suffix, I adapted that one.
https://github.com/pipe-cd/pipecd/blob/master/pkg/config/analysis_template.go#L33
https://github.com/pipe-cd/pipecd/blob/master/pkg/config/event_watcher.go#L62

Reply:
I think without suffix is better in that case since `config.LoadApplication` looks natural, but I understand your feel 😄 I thought the same (LoadApplicationConfig) at first but config.LoadXXXConfig looks redundant anyway.

Reply:
Thanks.

Happy to see your thoughts.

So let's go with it.